_includes/programs.html

```diff
@@ -118,11 +116,11 @@
       href="/programs/communitybridge/2020"
```

Comment:
Link needs to be updated as above

Reply:
after updating the link it shows this 

<img width="1438" alt="Screenshot 2021-09-03 at 6 41 56 PM" src="https://user-images.githubusercontent.com/81188792/132010652-c663d5b1-1e25-4a2e-bbe1-2e54e1089fa3.png">